dash/components/src/Settings/NotificationMethod.tsx

```diff
@@ -19,35 +20,46 @@ const NotificationMethod: React.FC<Props> = ({
   inUse,
   createNotification,
 }) => (
-  <li className="pl-4 xs:pr-2 py-2 odd:bg-slate-100 xs:odd:bg-slate-50 rounded-lg max-w-4xl flex justify-between items-center">
-    <div className="flex flex-col">
-      <h2 className="text-slate-700">
-        <span className="capitalize">{method}</span>
-        <span className="text-violet-700 font-medium pl-1.5">{prettyE164(value)}</span>
+  <li className="pl-4 xs:pr-2 py-2 odd:bg-slate-100 xs:odd:bg-slate-50 rounded-xl max-w-4xl flex flex-row justify-between items-center group">
+    <div className="flex-grow overflow-hidden pl-2 -ml-2 relative">
+      {!inUse && <span className="font-semibold text-red-500 text-sm">Not in use</span>}
+      <h2 className={cx(!inUse && `opacity-30`, `flex items-center`)}>
+        <div
+          className={cx(
+            `w-2 h-2 rounded-full mr-2 relative shrink-0`,
+            inUse ? `bg-violet-500` : `bg-slate-400/70`,
+          )}
+        >
+          <div
+            className={cx(
+              `w-2 h-2 bg-violet-500 rounded-full absolute left-0 top-0 animate-ping`,
```

Comment:
love how this looks, but it's going to kill us in Argos.  can you use the `isScreenshotTest()` function to only add this class (`animate-ping`) when we're not screenshot testing?